cpp/include/cuml/cuml.hpp

```diff
@@ -19,105 +19,10 @@
 #include <cuda_runtime.h>
 #include <cuml/common/cuml_allocator.hpp>
 #include <memory>
+#include <raft/handle.hpp>
```

Comment:
this now means that we'll have to update our cmake install target to also include raft headers in our distribution of libcuml++.